packages/scaffold/src/views/w3m-connecting-siwe-view/index.ts

```diff
@@ -0,0 +1,110 @@
+import { CoreHelperUtil, OptionsController, RouterController, StorageUtil } from '@web3modal/core'
+import { customElement } from '@web3modal/ui'
+import { LitElement, html } from 'lit'
+import { state } from 'lit/decorators.js'
+
+@customElement('w3m-connecting-siwe-view')
+export class W3mConnectingSiweView extends LitElement {
+  // -- Members -------------------------------------------- //
+  private usubscribe: (() => void)[] = []
+
+  // -- State & Properties --------------------------------- //
+  @state() private dappIcons = OptionsController.state.metadata?.icons
```

Comment:
These can be non-state / readonly variables. They are not expected to change once dapp has initialized web3modal. Thus, no subscribe / unsubscribe is needed either.